modules/page/html/render/public.js

```diff
@@ -259,6 +262,24 @@ exports.create = function (api) {
       ]
     }
 
+    function noVisibleNewPostsWarning() {
```

Comment:
this is mostly repeated from modules/page/html/render/channel.js, is it possible (and desirable) to extract out the similar bits into a common module?

Reply:
At the time, and thought that trying to abstract it maybe didn't benefit us much because I'd just be replacing 'when' with another function call. However, now that you mention it, I notice that most of the HTML is shared too.

I guess the explanation of why the warning is being shown 'You won't see new public posts' or 'you won't see new channel posts' can be parameterised, which would make it more worthwhile.

Next time I work on the branch, I'll push something like: 

`function renderWarning(conditionObs, explanation) {
          var content =
        h('div', {classList: 'NotFollowingAnyoneWarning'}, h('section -notFollowingAnyoneWarning', [
          h('h1', i18n('You are not Following Anyone')),
          h('p', explanation)),
          h('p', [i18n("For help, see the 'Getting Started' guide at "),
            h('a', {href: 'https://scuttlebutt.nz/getting-started.html'}, 'https://scuttlebutt.nz/getting-started.html')]
          ),
        ]))

      return when(conditionObs, content)
}`

I'll check out the mixin thing you linked me too =].

Cheers for the review :D.